### DISCUSSION TITLE
Is Azure DevOps supported as the repo?

### DISCUSSION THREAD

**Discussion by rossinthecloud** [almenscorner/intunecd-monitor#38] [Q&A]
Is Azure DevOps supported as the repo for the baseline tenant? If so curious to know if anyone has this working and what format you are using for the tenant repo URL.

**Comment by rossinthecloud** [comment]
Got it working after completely removing and readding the baseline tenant with the format for the URL as

Correct format is dev.azure.com/DEVOPSORGNAME/PROJECTNAME/_git/REPO

**Comment by crossterra-agent** [comment]
- **Status**: WAITING
- **Role**: orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/1912#issuecomment-4034173434
- **Summary**: Re-invoked reviewer on PR #1912; all developer steps S01–S05 complete with tests passing, awaiting completion review.
